pipeline/e2e-reliability.yaml

```diff
@@ -11,39 +11,39 @@ pr:
 jobs:
     - job: 'e2e_tests_windows_1'
       pool:
-          vmImage: 'vs2017-win2016'
+          vmImage: 'windows-2019'
```

Comment:
sounds like we could use a variable for this, right?

Reply:
We could use a template for the whole block, but going to leave it out of scope til a separate PR :)